packages/siop-oid4vp/lib/rp/RP.ts

```diff
@@ -133,6 +140,32 @@ export class RP {
     })
   }
 
+  static async processJarmAuthorizationResponse(
+    response: string,
+    opts: {
+      decryptCompact: (input: {
+        jwk: { kid: string }
```

Comment:
Is there a need for the kid to be required? Kids are application specific and there is no guarantee they are present

Reply:
This is from Jarm. It's the 'default' approach.
I can also make the kid optional however, but I think for now we will always use this approach.

(OPTIONAL) The client decrypts the JWT using the key determined by the kid JWT header parameter. The key might be a private key, where the corresponding public key is registered with the expected issuer of the response ("use":"enc" via the client's metadata jwks or jwks_uri) or a key derived from its client secret (see section 4.2).